packages/components/pie-tag/test/component/pie-tag.spec.ts

```diff
@@ -12,18 +12,17 @@ type VariantToBgStyle = {
 
 const variantsToIsStrongStyle: Array<VariantToBgStyle> = [
```

Comment:
Do we need to update the visual tests?

Reply:
the visual test variants are dynamic so updating the values in defs.ts should reference the new values automatically